src/intl/en.json

```diff
@@ -155,5 +155,120 @@
+  "page-what-is-ethereum-banking-card": "Banking for everyone",
+  "page-what-is-ethereum-banking-card-desc": "Not everyone has access to financial services. But all you need to access Ethereum and its lending, borrowing and savings products is an internet connection.",
+  "page-what-is-ethereum-internet-card": "A more private internet",
+  "page-what-is-ethereum-internet-card-desc": "You don't need to provide all your personal details to use an Ethereum app. Ethereum is building an economy based on value, not surveillance.",
+  "page-what-is-ethereum-p2p-card": "A peer-to-peer network",
+  "page-what-is-ethereum-p2p-card-desc": "Ethereum allows you to move money, or make agreements, directly with someone else. You don't need to go through intermediary companies.",
+  "page-what-is-ethereum-censorless-card": "Censorship-resistant",
+  "page-what-is-ethereum-censorless-card-desc": "No government or company has control over Ethereum. This decentralization makes it nearly impossible for anyone to stop you from receiving payments or using services on Ethereum.",
+  "page-what-is-ethereum-commerce-card": "Commerce guarantees",
+  "page-what-is-ethereum-commerce-card-desc": "Ethereum creates a more level playing field. Customers have a secure, built-in guarantee that funds will only change hands if you provide what was agreed. You don’t need large company clout to do business.",
+  "page-what-is-ethereum-compatibility-card": "Compatibility for the win",
+  "page-what-is-ethereum-compatibility-card-desc": "Better products and experiences are being built all the time because Ethereum products are compatible by default. Companies can build on each other's success.",
+  "page-what-is-ethereum-native-title": "ETH",
+  "page-what-is-ethereum-native-alt": "The symbol for Ether (ETH)",
+  "page-what-is-ethereum-native-crypto": "Ethereum's native cryptocurrency and equivalent to Bitcoin. You can use ETH on Ethereum applications or for sending value to friends and family.",
+  "page-what-is-ethereum-wallets": "Wallets",
+  "page-what-is-ethereum-native-img-alt": "An illustration of a robot with a safe for a torso, used to represent Ethereum wallets",
+  "page-what-is-ethereum-wallets-desc": "How you manage your ETH and your Ethereum account. You'll need a wallet to get started – we'll help you choose one.",
+  "page-what-is-ethereum-dapps-title": "Ethereum dapps",
+  "page-what-is-ethereum-dapps-img-alt": "An illustration of a doge using an Ethereum application on a computer",
+  "page-what-is-ethereum-dapps-desc": "Products and services that run on Ethereum. There are dapps for finance, work, social media, gaming and more – meet the apps for our digital future.",
+  "get-eth-local-cryptos": "Localcryptos.com",
+  "get-eth-1inch": "1inch",
+  "get-eth-bancor": "Bancor",
+  "get-eth-dYdX": "dYdX",
+  "get-eth-Kyber": "Kyber",
+  "get-eth-Loopring": "Loopring",
+  "get-eth-Uniswap-v1": "Uniswap V1",
+  "get-eth-how-to-buy-meta-title": "How to buy ETH",
```

Comment:
Let's follow the pattern of the other pages... i.e. `page-get-eth-meta-title` & `page-get-eth-meta-description`